py-polars/polars/functions/lazy.py

```diff
@@ -127,26 +149,76 @@ def count(column: str | None = None) -> Expr:
     ╞═════╡
     │ 2   │
     └─────┘
+
+    Return the number of non-null values in multiple columns.
+
+    >>> df.select(pl.count("b", "c"))
+    shape: (1, 2)
+    ┌─────┬─────┐
+    │ b   ┆ c   │
+    │ --- ┆ --- │
+    │ u32 ┆ u32 │
+    ╞═════╪═════╡
+    │ 1   ┆ 3   │
+    └─────┴─────┘
+
     """  # noqa: W505
-    if column is None:
+    if columns is None:
+        if more_columns:
+            raise ValueError(
+                "cannot set `columns` to None while also providing positional arguments"
+            )
         return wrap_expr(plr.count())
 
-    return F.col(column).count()
+    columns = combine_column_names(columns, *more_columns)
+    return F.col(columns).count()
 
 
-def implode(name: str) -> Expr:
+@deprecate_renamed_parameter("name", "columns", version="0.20.4")
+def implode(columns: str | Sequence[str], *more_columns: str) -> Expr:
```

Comment:
I don't really see the point of using the `columns` / `*more_columns` approach here. These are intended as shortcuts, so who is really writing `implode(column="foo")` when they could write `col("foo").implode()`?

Just having a `*columns: str` signature is fine here, in my opinion. The 'more columns' thing is a bit confusing and I'm not too fond of it, though it has its benefits elsewhere.

That makes parsing everything a lot easier too - won't even need a util I think.

Reply:
The two-arg approach matches what we have in `select` & co., so it's consistent with what we do elsewhere; however a single `*columns` is definitely cleaner. If you're ok with it, I'll make the change ;)

Reply:
For these 'shortcut' functions I strongly prefer the `*columns` variant. Keep 'm clean and lean!

We could do it without breaking the existing keyword arg syntax by having a `*columns, old_arg=None` signature, check for usage of the old arg and throw a deprecation warning.

Reply:
(Lol; moved my reply to the top-level; but yes, I'm in favour of this ;)